workshops/chat/README.md

```diff
@@ -0,0 +1,258 @@
+# Chat
+
+Short link to this workshop: [https://workshops.hackclub.com/chat](https://workshops.hackclub.com/chat)
+
+---
+
+In this workshop we'll be implementing a real-time chat using Firebase. This workshop is a revival of the nowdefunct tutorial made by Firebase itself.
+
+**Table of Contents**
+
+- Part I: Set-up
+- Part II: The HTML File
+- Part III: The JS File
+- Part IV: Publishing and Sharing (and Chatting!)
+- Part V: Hacking
+
+## Part I: Set-up
+
+- Open up your C9 workspace
+- Make a new folder and name it `chat`
+- Make a new file within `chat` and name it `index.html`
+- Make another new file within `chat` and name it `main.js`
+- Add a reference to Firebase, by adding this to the **body** of `index.html`:
+
+  ```html
+  <script src="https://www.gstatic.com/firebasejs/live/3.0/firebase.js"></script>
+  ```
+
+- Add a reference to jQuery,by adding this after the previous line:
```

Comment:
This can be rephrased to be simpler. Perhaps just "Now let's also add jQuery. Insert the following script tag below Firebase's."

Have we introduced jQuery yet? I forget if we use it in That Was Easy.